### DISCUSSION TITLE
disable plugin on only one device

### DISCUSSION THREAD

**Discussion by droogi** [LMS-Community/slimserver#849] [Q&A]
I do not find any option: i would like to disable spotty plugin on one device. It would be good with a timer. So no spotify between 8pm and 14pm on my sons squeezebox radio. Is there a posibility how i could do that?
Regards droogi

**Comment by michaelherger** [comment]
I'm sorry, that's not possible. I've been thinking about something like that for my kids again and again... I tried a few things, but decided I'd try to manage the situation discussing the issue with them instead. I doubt I saved any time 😁.
